Upstream: not upstreamable
Reason: disable building of a test which needs a running X server

From 117f71dc86c5d492598e65df49de96ea91a91253 Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Sat, 29 Mar 2014 18:22:43 +0100
Subject: [PATCH] autotests: Don't build kpluginfactorytest

It needs a running X server
---
 autotests/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/autotests/CMakeLists.txt b/autotests/CMakeLists.txt
index dcc4a40..053b143 100644
--- a/autotests/CMakeLists.txt
+++ b/autotests/CMakeLists.txt
@@ -37,14 +37,12 @@ target_link_libraries(multiplugin Qt5::Widgets)
 
 kservice_unit_tests(
  kautostarttest
- kpluginfactorytest
  kpluginloadertest
  ksycocadicttest
  ksycocathreadtest
  kservicetest
  pluginlocatortest
 )
-target_link_libraries(kpluginfactorytest Qt5::Widgets)
 
 target_compile_definitions(kpluginloadertest PRIVATE
     FAKEPLUGIN_FILE="$<TARGET_FILE:fakeplugin>"
-- 
1.9.1

